tests/microgrid/test_grid.py

```diff
@@ -64,7 +36,9 @@ async def test_grid_2(mocker: MockerFixture) -> None:
     components = {grid_1, meter_2}
     connections = {ComponentConnection(source=grid_1.id, destination=meter_2.id)}
 
-    graph = gr._MicrogridComponentGraph(  # pylint: disable=protected-access
+    graph = gr.ComponentGraph[
+        Component, ComponentConnection, ComponentId
+    ](  # pylint: disable=protected-access
```

Comment:
Not needed anymore, right?
```suggestion
    ](
```

Reply:
Done here: a076b683afa74f6e0c8fa6b8d1ad1aa7c3f9eab4